src/app/screens/HomeScreen/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { FlatList, ListRenderItem } from 'react-native';
+
+import styles from './styles';
+import { DATA } from './constants';
+
+import BookDetail from '@app/components/BookDetail';
+import { Book } from 'src/Interfaces/Book';
```

Comment:
Move these two imports below global imports:
```
import React from 'react';
import { FlatList, ListRenderItem } from 'react-native';
import BookDetail from '@components/BookDetail';
import { Book } from '@interfaces/book';

import styles from './styles';
import { DATA } from './constants';
```

Reply:
I tried it but eventually the prettier ends up setting it like this

Reply:
My bad, bad linter config